deploy-as-code/helm/charts/ifix-services/adapter-master-data-service/templates/ingress.yaml

```diff
@@ -0,0 +1,2 @@
+# ingress.yaml
+{{- template "common.ingress" . -}}
```

Comment:
We need ingress only when there are api calls coming from outside the kube cluster.